src/App.js

```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
+import Calculator from './components/Calculator';
 import './App.css';
 
 function App() {
   return (
     <div className="App">
```

Comment:
- [ ] It is a requirement that `app.js` should be a class based component, kindly make this change